src/main/java/dk/cs/aau/huppaal/controllers/HUPPAALController.java

```diff
@@ -239,6 +240,24 @@ public void onChanged(final Change<? extends Component> c) {
 
         initializeReachabilityAnalysisThread();
 
+        File serverFile = UPPAALDriver.getServerFile();
+        StringBuilder sb = new StringBuilder("The file: '" + serverFile.getName() + "' does not exist at location: '" + serverFile.getAbsolutePath() + "'");
+
+        int i = 0;
```

Comment:
Could you add a comment about what this does?

Reply:
(The lines below what is shown)

Reply:
I have moved the stringbuilder and tooltip into an if, so that it is not shown when the file exists. I have added a comment above the if and then above the if statement inside the if. The inner if is added to remove potential space at the beginning of the new line

Reply:
super, you might consider to a some point to move this to a function

Reply:
Yes, I'm planning on doing the check in another way in the Singleton implementation of the UPPAALDriver, so it is just the temporary fix